js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js

```diff
@@ -508,6 +528,11 @@
               '',
               true,
               [
+                ...(!extendedNames ? traditionalNameValidation : extendedValidation),
+                {
+                  rule: Joi.string().max(256, 'utf8'),
+                  msg: 'Collection name max length is 256.'
```

Comment:
Max collection name length is 256 bytes, not 256 UTF-8 characters (which could use more than 256 bytes storage to represent them)

Reply:
Yes, you are right. I verified this
1. For Joi: since we provided the encoding, Joi does use the [byte length](https://github.com/hapijs/joi/blob/master/lib/types/string.js#L471), ([docs](https://github.com/hapijs/joi/blame/master/lib/types/string.js#L471), [issue](https://github.com/hapijs/joi/blame/master/lib/types/string.js#L471)). We have to add normalize() too
2. For Yup, I am adding a custom validation method which will do this
3. For JSONs (i.e. edits), we are currently not doing length validation